### PR TITLE
People who punch the soul rambler now say things

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
Shout-out to chungus aka my boy coxswain
They say either "Take that!" or "Taste the pain!"
:cl:
 * rscadd: People that punch the Soul Rambler now say things.

### CODE REVIEW

**Comment by SonixApache** [comment]
🤔

**Comment by Kurfursten** [comment]
Nice

**Comment by SkowronX** [comment]
?

**Comment by aceedex** [comment]
truly, barry is our most valuable asset

File: code/modules/mob/living/carbon/human/combat.dm [hunk 1]
```diff
@@ -205,6 +205,9 @@
 		if(prob(chance))
 			knock_out_teeth(attacker)
 
+	if(isrambler(src) && ishuman(attacker))
+		attacker.say(pick("Take that!", "Taste the pain!"))
```
Comment (jknpj) [inline code/modules/mob/living/carbon/human/combat.dm:209]: make this a list

Comment (DamianX) [reply code/modules/mob/living/carbon/human/combat.dm:209]: I think it's fine

Comment (ShiftyRail) [reply code/modules/mob/living/carbon/human/combat.dm:209]: Maybe make it a probabilistic roll

**Comment by SirSkeleto** [comment]
dissenters please note that this only occurs for punching

**Comment by Gringonius** [comment]
Needs to work on kicking too.

**Comment by Kurfursten** [comment]
Add it to the kick_act too, found in human_attackhand.dm L73

**Pull request merged**